globus_cli/config.py

```diff
@@ -94,6 +110,30 @@ def get_transfer_refresh_token():
     return lookup_option(TRANSFER_RT_OPTNAME)
 
 
+def get_auth_access_token():
```

Comment:
Is there a particular reason that this is still separate from loading the refresh token? (Likewise for `get_transfer_access_token()`)
It feels like a development decision that's carrying over.

Assuming there isn't a strong reason, could you fold these together into something like `get_auth_credentials`?
It's fewer names to export/import/keep-track-of.